Functions/BIDS/xASL_bids_parms2BIDS.m

```diff
@@ -345,6 +345,10 @@
     outParms = xASL_io_CheckDeprecatedFieldsX(outParms);
 end
 
+% If we convert to legacy, we make sure to properly initialize the field Q.Sequence
```

Comment:
Also, does xASL_bids_parms2BIDS now move quantification fields like Sequence, PLD, etc to JSON.Q.fields or to JSON.fields? Because BIDS is JSON.fields...

So perhaps we should add either x.Q.Sequence or JSON.Sequence. But not JSON.Q.Sequence because we do this for none of the parameters right?

Reply:
1. I've changed that to outParms.Q.Sequence. While it will end up in x.Q.Sequence, in theory, we just provide output as outParms and it's up to the user to add it to the x-struct or not. 
2. parms2BIDS does both BIDS->Legacy and Legacy->BIDS. So when doing Legacy->BIDS, it is all in JSON.fields. When doing BIDS->Legacy, some fields are in JSON.fields and some in JSON.Q.fields, depending on the Legacy format. Note that we don't use x or JSON, we call it outParms for simplicity.

I hope this makes sense. I've improved the comments.